library/src/scala/quoted/Quotes.scala

```diff
@@ -3428,6 +3428,23 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
       def unapply(x: NoPrefix): true
     }
 
+    // ----- Flexible Type --------------------------------------------
+
+    /** Flexible types for explicit nulls */
+    type FlexibleType <: TypeRepr
```

Comment:
All these definitions should be annotated with `@experimental`. The documentation in of `reflectModule` must also be updated.

Reply:
Flexible types are not considered experimental, though. So how can a non-experimental macro deal with all possible types it manipulates if we make `FlexibleType` experimental?

Reply:
Oh, then the MiMa filters must be updated.

Reply:
MiMa filters updated